### DISCUSSION TITLE
Gratitude

### DISCUSSION THREAD

**Discussion by StephenCWLL** [tom-james-watson/breaktimer-app#91] [General]
I just wanted to thank the programmer for this software which I've started using after just having severe eye strain. Was pleased to find this simple reliable free software.

Multiple breaks would be a good addition too but keep up the good work :)

**Comment by tom-james-watson** [comment]
My pleasure - glad you're enjoying it :smile:

**Comment by jim9835** [comment]
Yes, good call. Thanks Tom, I have been using this for ages, works a treat.

**Comment by danielaj28** [comment]
Thank you @tom-james-watson your app is great and have been using it for months. Now that I have found out it is open source i will be looking to contribute where i can!

**Comment by camilogarciabotero** [comment]
I join the previous comments: thank you @tom-james-watson and all the developers!
